client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -95,6 +96,7 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
                 <Grid item>
                     <FormInput fieldName='קשר'>
                         <AlphanumericTextField
+                            testId='relationship'
```

Comment:
Shouldn't be test-id?

Reply:
Where it is wrapped it needs testId, in places it is not wrapped it needs test-id